Backend/templates/csvmail.py

```diff
@@ -0,0 +1,51 @@
+#mail template to send password to user
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+
+import configparser
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+URL = config['Server']['domain']
+
+def get_message(name: str, password: str, mail: str) -> str:
+    
+    msg = MIMEMultipart('alternative')
+    msg.set_charset("utf-8")
+    msg['Subject'] = 'Dein Passwort für das Abstimmtool'
+    msg['From'] = config['Mail_Server']['mail']
+    msg['To'] = mail
+
+    # Attach parts into message container.
+    # According to RFC 2046, the last part of a multipart message, in this case
+    # the HTML message, is best and preferred.
+
+    html = '''

```

Comment:
eher ein future issue (ist auch mein code, i know :D), vllt können wir das html in ein eigenes .html file auslagern, das erlaubt dann auch mehr customization und zusätzlich auch ein hübsches preview + Separation of concerns